handlers/challenge_handler.py

```diff
@@ -854,6 +856,13 @@ def update_ctf_purpose(slack_wrapper, ctf):
 
         slack_wrapper.set_purpose(ctf.channel_id, purpose)
 
+    @staticmethod
+    def set_ctf_topic(slack_wrapper, ctf):
```

Comment:
This should just take a string, instead of a `ctf` object.

Reply:
Or, if you want it to be ctf cred specific, I couldn't create this function, and instead make the call to `set_topic` directly in your above part.